### DISCUSSION TITLE
"|" not set as is_punct

### DISCUSSION THREAD

**Discussion by lise-brinck** [explosion/spaCy#11289] [Language Support]
In the Norwegian and Danish models, "|" is not set as is_punct. The same seems to be [the case for other languages as well](https://stackoverflow.com/questions/56019093/how-can-i-add-custom-signs-to-spacys-punctuation-functionality). I can't think of a case where this character shouldn't be tagged as punctuation. Should it not be in the list of punctuation?

**Comment by adrianeboyd** [comment]
`is_punct` is using this definition based on unicode categories:

https://github.com/explosion/spaCy/blob/ed4ad309e6dd6fb420cbf18e4fd5e8de3291eeba/spacy/lang/lex_attrs.py#L26-L30

It looks like `|` is considered a math symbol instead in the unicode categories.
